eZ/Publish/API/Repository/ContentService.php

```diff
@@ -432,4 +432,15 @@ public function newTranslationInfo();
      * @return \eZ\Publish\API\Repository\Values\Content\TranslationValues
      */
     public function newTranslationValues();
+
+    /**
+     * Returns true if the given content type $contentType has content instances.
+     *
+     * @since 6.0.1
+     *
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return bool
+     */
+    public function isContentTypeUsed(ContentType $contentType);
```

Comment:
maybe... issue says content type removal doesn't take in account content in trash so we're adding isContentTypeUsed method, right? But can't ContentType be used with policies as sections can? Shouldn't ContentType take also this is account? (maybe that's another issue though)